### PR TITLE
IDEMPIERE-5838 Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Fixes user-facing and non-user-facing typos. Found via `codespell -q 3 -S "*.map,./migration,./migration-historic"`

https://idempiere.atlassian.net/browse/IDEMPIERE-5838

# Pull Request Checklist

- [X] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [X] My code follows the best practices of this project
- [ ] I have performed a self-review of my own code
- [ ] My code is easy to understand and review. 
- [ ] I have checked my code and corrected any misspellings
- [ ] In hard-to-understand areas, I have commented my code.
- [ ] My changes generate no new warnings
### Tests
- [ ] I have tested the direct scenario that my code is solving
- [ ] I checked all collaterals that can be affected by my changes, and tested other potential affected scenarios
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added unit tests that prove my fix is effective or that my feature works
### Documentation
- [X] I have made corresponding changes to the documentation as follows:
- - [ ] New feature (non-breaking change which adds functionality): I have created the New Feature page in the project wiki explaining the functionality and how to use it. If relevant, I have committed sample data to the core seed to have usable examples in GardenWorld.
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected): I have documented the change in a clear way that everyone in the community can understand the impact of the change.
- - [ ] Improvement (improves and existing functionality): This documentation is needed if the improvement changes the way the user interacts with the system or the outcome of a process/task changes. If it is just, for instance, a performance improvement, documentation might not be needed. 
- [ ] The changed/added documentation is in the project wiki (not privately-hosted pdf files or links pointing to a company website) and is complete and self-explanatory.

### CODE REVIEW

**Comment by hengsin** [comment]
Is there a jira ticket created for this ? Every pull request must have a corresponding jira ticket at https://idempiere.atlassian.net/

**Comment by luzpaz** [comment]
No i didn't set up a JIRA account. Will you make an exception ?

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/form/WTabEditor.java [hunk 1]
```diff
@@ -127,9 +127,9 @@ public class WTabEditor extends TabEditor implements IFormController, EventListe
 	protected WYesNoEditor editorIsReadOnly = null;
 
 	//Child of westVLayout
-	/** List of display fields */
+	/** List of visible display fields */
 	private Listbox visible = new Listbox();
-	/** List of not dispaly fields */
+	/** List of invisible display fields */
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/form/WTabEditor.java]: How about '"List of undisplayed fields" ?

File: org.adempiere.base/src/org/compiere/model/I_U_POSTerminal.java [hunk 2]
```diff
@@ -133,12 +133,12 @@ public interface I_U_POSTerminal
     public static final String COLUMNNAME_CashBookTransferType = "CashBookTransferType";
 
 	/** Set Cash Book Transfer Type.
-	  * Where the money in the cash book should be transfered to. Either a Bank Account or another Cash Book
+	  * Where the money in the cash book should be transferred to. Either a Bank Account or another Cash Book
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply org.adempiere.base/src/org/compiere/model/I_U_POSTerminal.java:136]: Should i revert of just keep it even if it gets overridden ?

**Comment by CarlosRuiz-globalqss** [comment]
> Is there a jira ticket created for this ? Every pull request must have a corresponding jira ticket at https://idempiere.atlassian.net/

Hi @hengsin - I think we can associate this ticket to [IDEMPIERE-5838](https://idempiere.atlassian.net/browse/IDEMPIERE-5838) - "Improve readability of code" - is that OK?

File: org.adempiere.base/src/org/compiere/model/I_U_POSTerminal.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (CarlosRuiz-globalqss) [reply org.adempiere.base/src/org/compiere/model/I_U_POSTerminal.java:136]: > Should i revert of just keep it even if it gets overridden ?

just keep it - I'm fixing it in another ticket
thanks for the tip about codespell

File: org.apache.ecs/src/org/apache/ecs/Attributes.java [hunk 3]
```diff
@@ -111,12 +111,12 @@ public interface Attributes
...
 
     /**
-        Do we surround attributes with qoutes?
+        Do we surround attributes with quotes?
     */
     public boolean getAttributeQuote();
 
     /**
-        Set wether or not we surround the attributes with quotes.
+        Set weather or not we surround the attributes with quotes.
```
Comment (CarlosRuiz-globalqss) [inline org.apache.ecs/src/org/apache/ecs/Attributes.java:119]: here also must be "whether"

**Comment by hengsin** [comment]
> > Is there a jira ticket created for this ? Every pull request must have a corresponding jira ticket at https://idempiere.atlassian.net/
> 
> Hi @hengsin - I think we can associate this ticket to [IDEMPIERE-5838](https://idempiere.atlassian.net/browse/IDEMPIERE-5838) - "Improve readability of code" - is that OK?

Yes, that's fine for me.

File: org.adempiere.base/src/org/compiere/model/MAssetProduct.java [hunk 4]
```diff
@@ -27,7 +27,7 @@
 
 /**
  * Asset Product
- * @author Teo Sarca, SC ARHIPAC SERIVCE SRL
+ * @author
```
Comment (hengsin) [inline org.adempiere.base/src/org/compiere/model/MAssetProduct.java:30]: accidental removal of author

File: org.adempiere.base/src/org/compiere/model/MEntityType.java [hunk 5]
```diff
@@ -86,16 +86,16 @@ public static MEntityType get(Properties ctx, String entityType)
...
+    /**
+     * UUID based Constructor
+     * @param ctx  Context
+     * @param AD_EntityType_UU  UUID key
+     * @param trxName Transaction
+     */
+    public MEntityType(Properties ctx, String AD_EntityType_UU, String trxName) {
+        super(ctx, AD_EntityType_UU, trxName);
+    }
+
```
Comment (hengsin) [inline org.adempiere.base/src/org/compiere/model/MEntityType.java:98]: saw many of this probably white space only changes, what exactly have been changed here ?

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/form/WTabEditor.java [hunk 6]
```diff
@@ -127,9 +127,9 @@ public class WTabEditor extends TabEditor implements IFormController, EventListe
 	protected WYesNoEditor editorIsReadOnly = null;
 
 	//Child of westVLayout
-	/** List of display fields */
+	/** List of visible display fields */
 	private Listbox visible = new Listbox();
-	/** List of not dispaly fields */
+	/** List of undisplayed fields */
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/form/WTabEditor.java:132]: I think "invisible" is the more appropriate term to use here.

**Review by hengsin — CHANGES_REQUESTED** [review]

File: org.adempiere.base/src/org/compiere/model/MEntityType.java (hunk 5 above)
Comment (luzpaz) [reply org.adempiere.base/src/org/compiere/model/MEntityType.java:98]: I'm not sure why vscode touched those blocks of code. Maybe some auto-linting feature but not clear what it linted? Perhaps these blocks were  ^M terminated ? I've seen that before...but here it's not clear that is the issue.

Comment (CarlosRuiz-globalqss) [reply org.adempiere.base/src/org/compiere/model/MEntityType.java:98]: > I'm not sure why vscode touched those blocks of code. Maybe some auto-linting feature but not clear what it linted? Perhaps these blocks were `^M` terminated ? I've seen that before...but here it's not clear that is the issue.

Yes, the same happened to me when editing a file on github, it complained that the file had mixed EOL characters and it was going to be unified.

I checked one of the files and those lines have Linux EOL, while the rest of the file have Windows EOL.

**Review by CarlosRuiz-globalqss — APPROVED** [review]

**Comment by CarlosRuiz-globalqss** [comment]
Hi @hengsin - I think this is ready for the "Squash and merge".

Is that OK?

I think when clicking the "Squash and merge" button we need to add the ticket number IDEMPIERE-5838 to the commit message.

**Comment by hengsin** [comment]
> Hi @hengsin - I think this is ready for the "Squash and merge".
> 
> Is that OK?
> 
> I think when clicking the "Squash and merge" button we need to add the ticket number IDEMPIERE-5838 to the commit message.

Yes, ok to merge.

**Pull request merged**